src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +30,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("Data was successful fetched from DB. Params: userId = {}", userId);
```

Comment:
i guess here u shoud call products , and if u will call products u need to override method toString in product class/ and maybe u need to delete all comments ?

Reply:
I do not think logging DB is a good idea. What if DB will contain 1K lines? 10K? 100K and even more?